L1Trigger/L1TZDC/data/zdcLUT_HI_v0_1.txt

```diff
@@ -0,0 +1,4613 @@
+# ZDC LUT for HI running 2023, first pass implementation
```

Comment:
> L1Trigger/L1TZDC/data/zdcLUT_HI_v0_1.txt

As far as I know, this kind of additions to `data/` should be integrated via https://github.com/cms-data/ (creating a new repository "cms-data/L1Trigger/L1TZDC".

Reply:
same for `L1Trigger/L1TZDC/data/zdcLUT_HI_DUMMY.txt`

Reply:
Both are removed in latest commits (side note - should I click the resolve button for these points or should you as reviewers? or leave them as is - I defer to you)

Reply:
You can resolve them.

I think only the person that opened the PR or the admin of the whole repository can do it.

Reply:
Understood - going thru and resolving where I believe commits have addressed comments